src/base/i18n.cc

```diff
@@ -62,17 +62,28 @@ std::string locale;
 std::string localedir;
 std::string homedir;
 
+char* (*gettext_)(const char*) = gettext;
```

Comment:
On second thoughts, I think this will not catch all cases. This only intercepts plain `gettext` but I guess it does not work for `pgettext`, `ngettext` etc? (did not test it yet)

Reply:
Not sure where all these are located but I will try to find them.